eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -6,7 +6,7 @@ steps:
   - pwsh: |
       git clone https://github.com/Azure/azure-sdk-tools.git $(Build.SourcesDirectory)/tools_repo
       cd $(Build.SourcesDirectory)/tools_repo
-      git checkout 35ad98f821913eb0e8872f861ee60589b563c865
+      git checkout latest-stable-tools
```

Comment:
Is this a manual tag you created? Is there any reason you don't use a repo tag we create from our automation?

Reply:
Ones that get created by https://dev.azure.com/azure-sdk/internal/_build?definitionId=1696

Reply:
Yeah. I understood the request to swap to a tag was such that we can avoid hitting an issue like this in the future.

If we just use the auto-release tag like you mention, then we'll still hit the same issue where we have to manually update this tag.

Dozie pointed me at: https://dev.azure.com/azure-sdk/internal/_build?definitionId=1696

Which I was going to update to also update the latest-stable-tools flag.

Reply:
I'm not a big fan of using "latest" as that has its own set of potential problems, like things getting broken underneath us when that pointer updates and we didn't test all the usages. This is the same reason we don't use the "latest" OS agent images and also pin to specific versions of tools, which is to increase our chances of repeatable builds. For this particular case I'd suggest we use the versioned tag produced by that build as that is what we use in other places that consume the tools repo as a product. I agree it requires some manual work to update when needed but that makes the work very explicit and we have a chance to test the change.

That said in the long-term I'd like this particular usage to end up consuming a tools package instead of cloning the source and running the tool from source. IIRC we have an issue tracking that work.

Reply:
It can also cause inadvertent problems when servicing.

Reply:
Yeah fair. Alright I'll change this and clean up that tag from the repo.